ert/data/record/_transmitter.py

```diff
@@ -49,6 +51,22 @@ class RecordTransmitterType(Enum):
     shared_disk = auto()
 
 
+def _record_path_to_record_dict(
```

Comment:
This needs a `test_record_path_to_record_dict()` in `test_transmitters.py`

Reply:
(see 52d7147)

Reply:
Fully agree, looks nice, Will you append it as a stand alone commit?